apps/web/pages/booking/[uid].tsx

```diff
@@ -545,6 +545,14 @@ export default function Success(props: SuccessProps) {
                         </div>
                       </>
                     )}
+                    {bookingInfo?.responses?.title && typeof bookingInfo?.responses?.title === "string" && (
```

Comment:
At Cal we have a rule: Prefer ternaries over “&&” for React Components. Read the following article: https://www.crocoder.dev/blog/react-conditional-rendering/ 

Instead of `bookingInfo?.responses?.title` we can use either ternaries or simpler version `!!bookingInfo?.responses?.title` instead.